README.md

```diff
@@ -12,9 +12,9 @@ _Bad software is everywhere, and we're tired of it. Sentry is on a mission to he
 
 Gradle Plugin for Sentry Android
 ===========
-[![Build](https://github.com/getsentry/sentry-android-gradle-plugin/actions/workflows/test-matrix-agp-gradle.yaml/badge.svg)](https://github.com/getsentry/sentry-android-gradle-plugin/actions/workflows/test-matrix-agp-gradle.yaml)
```

Comment:
This was called build but was looking at the test-matrix workflow instead of the build workflow. Also adjusted it to point to the main branch.